wikipedia-sentences/README.md

```diff
@@ -66,59 +92,114 @@ shuf input.txt > input.txt
 
 ```sh
 python app.py -t query_restful
 ```
 
-Then:
+You should open another terminal window and paste the following command. 
+
+```sh
+curl --request POST -d '{"top_k": 5, "mode": "search",  "data": ["hello world"]}' -H 'Content-Type: application/json' 'http://0.0.0.0:45678/search'
+```
+
+Once you run this command, you should see a JSON output returned to you. This contains the five most semantically similar Wikipedia sentences to the text input you provided in the 'data' parameter. Feel free to alter the text in the 'data' parameter and play around with other queries! For a better understanding of the parameters see the table below. 
+|  |  |
+|--|--|
+| `top_k` | Integer determining the number of sentences to return |
+| `mode` | Mode to trigger in the call. See [here](https://docs.jina.ai/chapters/rest/) for more details |
+| `data` | Text input to query |
+ 
+
+
+
+
+### Using JinaBox; our frontend search interface
+
+**JinaBox** is a light-weight, highly customizable JavaScript based front-end search interface. To use it for this example, begin by opening the REST API interface. 
 
 ```sh
-curl --request POST -d '{"top_k": 10, "mode": "search",  "data": ["hello world"]}' -H 'Content-Type: application/json' 'http://0.0.0.0:45678/search'
-````
+python app.py -t query_restful
+```
 
-Or use [Jinabox](https://jina.ai/jinabox.js/) with endpoint `http://127.0.0.1:45678/search`
+In your browser, open up the hosted JinaBox on [jina.ai/jinabox.js](https://jina.ai/jinabox.js/). In the configuration bar on the left hand side, choose a custom endpoint and enter the following: `http://127.0.0.1:45678/search` . You can type in search queries into the text box on the right hand side!
 
-### From the Terminal
+
+### Directly from terminal
+You can also easily search (query) your data directly from the terminal. Using the following command will open an interface directly in your terminal window. 
 
 ```sh
 python app.py -t query
 ```
 
-## 👷 Build a Docker Image
+# Overview of the files in this example
+Here is a small overview if you're interested in understanding what each file in this example is doing. 
 
-This will create a Docker image with pre-indexed data and an open port for REST queries.
+`data/toy-input.txt` - contains a small number of sentences to test the example without downloading anything. `flows/index.yml` - contains the configuration details for indexing data. 
+`flow/query.yml` - contains the configuration details for querying data. 
+`pods/encode.yml` - specifies which executor should be used to encode the data. 
+`pods/index.yml` - specifies which executor should be used to index and store the data. 
+`test/*` - various maintenance tests to keep the example running. 
+`app.py`  - the gateway code to combine the index and query flow 
+`get_data.sh` - downloads the Kaggle dataset.
+`manifest.yml` - needed to deploy to Jina Hub
+`requirements.txt` - contains all required python libraries
```

Comment:
Might be most readable to do this as a table

Reply:
will make a pr for the template